msg/messages.js

```diff
@@ -1015,6 +1015,13 @@ Blockly.Msg.LISTS_SPLIT_TOOLTIP_SPLIT = 'Split text into a list of texts, breaki
 /// https://github.com/google/blockly/wiki/Lists#make-text-from-list] for more information.
 Blockly.Msg.LISTS_SPLIT_TOOLTIP_JOIN = 'Join a list of texts into one text, separated by a delimiter.';
 
+/// url - Information describing reversing a list.
+Blockly.Msg.LISTS_REVERSE_HELPURL = '';
```

Comment:
Go ahead and fill this in as: https://github.com/google/blockly/wiki/Lists#reversing-a-list

When you submit, also make a new issue to create this wiki content.  We can probably make you an editor for the wiki and would greatly appreciate help adding the docs for your new blocks there.

Unless you find a Wikipedia or similar description of reversing a list. That would be good, too.